tests/SonataPageAdminTraitTest.php

```diff
@@ -0,0 +1,675 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the behat/helpers project.
+ *
+ * (c) Ekino
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tests\Ekino\BehatHelpers;
+
+use Behat\Mink\Driver\DriverInterface;
+use Behat\Mink\Element\DocumentElement;
+use Behat\Mink\Element\NodeElement;
+use Behat\Mink\Exception\ElementNotFoundException;
+use Behat\Mink\Session;
+use Behat\Mink\WebAssert;
+use Ekino\BehatHelpers\SonataPageAdminTrait;
+use phpmock\Mock;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Twig\Node\EmbedNode;
+
+/**
+ * @author William JEHANNE <william.jehanne@ekino.com>
+ */
+class SonataPageAdminTraitTest extends TestCase
+{
+    private $trait;
```

Comment:
missing phpdoc

Reply:
if I add phpDoc here, phpstan fail

Reply:
I'll check how & why :)

Reply:
I don't understand for the moment, but this is the error : 
```------ ----------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Line   tests/SonataPageAdminTraitTest.php
 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------------------
  36     Property Tests\Ekino\BehatHelpers\SonataPageAdminTraitTest::$trait has invalid type Ekino\BehatHelpers\SonataPageAdminTrait.
  88     Property Tests\Ekino\BehatHelpers\SonataPageAdminTraitTest::$trait (Ekino\BehatHelpers\SonataPageAdminTrait&PHPUnit\Framework\MockObject\MockObject) does not accept
         PHPUnit\Framework\MockObject\MockObject.
 ------ ----------------------------------------------------------------------------------------------------------------------------------------------------------------------```

Reply:
ok my bad. defining it as a property was a bad idea i guess. Can you uniformize as it is done in other tests without defining it as a property of the class plz?